CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ INCLUDE(CheckIncludeFileCXX)
 #set(CMAKE_C_COMPILER clang)
 #set(CMAKE_CXX_COMPILER clang++)
 
+if(APPLE)
+    set(OPENSSL_ROOT_DIR /usr/local/opt/openssl)

```

Comment:
Looks like a hardcode. Is it always there?

Reply:
Yes, it's always there.

It's from an official generator of TDLib builder - https://tdlib.github.io/td/build.html?language=JavaScript